testbench/common.py

```diff
@@ -814,10 +823,22 @@ def handle_retry_test_instruction(database, request, socket_closer, method):
                 "Injected 'connection reset by peer' fault", 500
             )
         elif items[0] == "broken-stream":
+            print("broken stream")
             return __get_streamer_response_fn(database, method, socket_closer, test_id)
     broken_stream_after_bytes = (
         testbench.common.retry_return_broken_stream_after_bytes.match(next_instruction)
     )
+
+    retry_stall_beginning_matches = testbench.common.retry_stall_beginning.match(
+        next_instruction
+    )
+    if retry_stall_beginning_matches and method == "storage.objects.get":
```

Comment:
next_instruction will have context on the retry_stall and the method so you don't need to check it again unless you're safe guarding so it's only supported for storage.objects.get? If you are, i would recommend relaxing it to support any operations.